scene/resources/font.cpp

```diff
@@ -1551,7 +1551,7 @@ void Font::draw_string(RID p_canvas_item, const Point2 &p_pos, const String &p_t
 	buffer->draw(p_canvas_item, ofs, p_modulate);
 }
 
-void Font::draw_multiline_string(RID p_canvas_item, const Point2 &p_pos, const String &p_text, HAlign p_align, float p_width, int p_max_lines, int p_size, const Color &p_modulate, int p_outline_size, const Color &p_outline_modulate, uint16_t p_flags) const {
+void Font::draw_multiline_string(RID p_canvas_item, const Point2 &p_pos, const String &p_text, HAlign p_align, real_t p_width, int p_max_lines, int p_size, const Color &p_modulate, int p_outline_size, const Color &p_outline_modulate, uint16_t p_flags) const {
```

Comment:
I see here you're changing `float` to `real_t` for the width parameter, but in TextServer it's changed the other way around. How do you decide? (In both cases it's small numbers a very high precision pixel values don't really make sense so `float` would be good enough precision wise.)

Reply:
I was going for the minimum to make it compile, but it's better to just make this float.